local_devnet/scripts/init.sh

```
@@ -5,21 +5,21 @@
-CELESTIA_HOME=$($CELESTIA_BIN config home)
-if [ -d "$CELESTIA_HOME" ]; then rm -r $CELESTIA_HOME; fi
+CELESTIA_APP_HOME=$($CELESTIA_BIN config home)
+if [ -d "$CELESTIA_APP_HOME" ]; then rm -r $CELESTIA_APP_HOME; fi
 $CELESTIA_BIN config set client chain-id local_devnet
 $CELESTIA_BIN config set client keyring-backend test
 $CELESTIA_BIN config set app api.enable true
 $CELESTIA_BIN keys add alice
 
 if [ "$MULTIPLEXER" = "true" ]; then
-cd $SCRIPT_DIR; cp ../celestia-app/genesis_046.json $CELESTIA_HOME/config/genesis.json; cd $PWD # use local_devnet genesis
+cd $SCRIPT_DIR; cp ../celestia-app/genesis_046.json $CELESTIA_APP_HOME/config/genesis.json; cd $PWD # use local_devnet genesis
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)